pom.xml.next

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should avoid using that command (and committing all the files without checking). I'd really recommend to not use idea for git operations, but that's up to you.